packages/wdio-allure-reporter/src/index.js

```diff
@@ -120,6 +124,20 @@ class AllureReporter extends WDIOReporter {
         currentTest.addLabel('thread', cid)
     }
 
+    getLabels(test){
+        var tags = test.tags
```

Comment:
I think would be better to use `let` or `const` instead of `var`

Reply:
Done